DEVELOPMENT.md

```diff
@@ -151,6 +151,83 @@ to start a debugger to which attach from your IDE.
    curl localhost:8081/apis/apps/v1/deployments
    ```
 
+## Database
+
+### Database requirements
+
+* psql
+    ```bash
+    $ sudo dnf install postgresql
+    $ psql --version
+    psql (PostgreSQL) 15.6
+    ```
+
+* Optional: A SQL client is recommended to interact easily with the database.
+  For example [DBeaver Community](https://dbeaver.io/). To configure DBeaver, check these 
+  [instructions](https://github.com/kubearchive/kubearchive/blob/main/database/README.md).
+
+
+### Test the connection to the Database.
+  
+1.  Identify the PostgreSQL pod.
+    ```bash
+    $ kubectl get pods -n [namespace]
+    NAME                        READY   STATUS    RESTARTS   AGE
+    postgresql-65db968757-ncnvk   1/1     Running   0          2m38s
+    ```
+1.  Replace the `postgres_pod`, `db_user` and `db_name` with the values from `ps-secret.yaml`, or from the `values.yaml` file. The `--password` prompts for the password interactively.
+    ```bash
+    $ kubectl exec -it [postgres_pod] -n [namespace] -- psql -h localhost -U [db_user] --password -p 5432 [db_name]
+    ```
+    ```bash
+    $ kubectl exec -it postgresql-65db968757-ncnvk -n kubearchive -- psql -h localhost -U ps_user --password -p 5432 postgresdb
+    Password: 
+    psql (16.3 (Debian 16.3-1.pgdg120+1))
+    Type "help" for help.
+
+    postgresdb=# 
+    ```
+
+### Create a Table and Populate the Database
+
+1.  Create a port-forward in a new terminal tab.
+    ```bash
+    $ kubectl port-forward -n [namespace] svc/postgres 5432:5432
+     ```
+1.  The script `database/init_db.go` will create a table in the database and will 
+    insert the test objects from `database/test_objects.sql`.
+    The script has as some default values for the `db_user`, `db_password`, and the `db_name`.
+    These values are the default ones from the `ps-secret.yaml` file. If you are
+    using others (defined in the `values.yaml` file) you can run the script with the following flags:
+    `--database-name`, `database-user`, and `database-password`. For more info you can run:
+    ```bash
+    $ go run init_db.go --help
+    ```
+    Run the script.
+    ```bash
+    $ cd database
+    $ go run init_db.go
+    table test_objects created in the DB.
+    testdata from test_objects.sql inserted in the DB.
+    ```
```

Comment:
I think this should be the first section before generating activity on the kubearchive sink. And I have a suggestion: as we added default values, we don't need to explain the credentials stuff.


```suggestion
### Create a Table and Populate the Database

1.  Create a port-forward in a new terminal tab.
    ```bash
    kubectl port-forward -n kubearchive svc/postgres 5432:5432
     ```
1. Run the script.
    ```bash
    cd database
    go run init_db.go
    ```
```

Reply:
Done